models/parrot_neuron.h

```diff
@@ -48,6 +48,10 @@ example a poisson generator with a high firing rate is connected
 to a parrot neuron, the communication cost associated with outgoing
 spikes is much bigger for the latter.
 
+Only spikes incoming on connections to port 0 will be repeated.
+Connections onto port 1 will be accepted, but spikes incoming
+through port 1 will be inignored.
```

Comment:
typo: inignored

I think it would be useful to briefly explain the purpose of allowing spike arrival via port 1 while ignoring those spikes.